.ci/install_kata_image.sh

```diff
@@ -6,56 +6,128 @@
 #
 
 set -o errexit
-set -o nounset
```

Comment:
could you try to not remove this, but instead define any variable that could be used?